addons/html_builder/static/tests/snippets_menu.test.js

```diff
@@ -17,6 +36,25 @@ test("open SnippetsMenu and discard", async () => {
     expect(".o_menu_systray .o-website-btn-custo-primary").toHaveCount(1);
 });
 
+test("navigate between builder tab don't fetch snippet description again", async () => {
+    onRpc("render_public_asset", () => {
+        expect.step("render_public_asset");
+    });
+    await setupWebsiteBuilder(`<h1> Homepage </h1>`);
+    await openSnippetsMenu();
+    expect(queryAllTexts(".o-website-snippetsmenu .o-snippets-tabs span")).toEqual([
+        "BLOCKS",
+        "CUSTOMIZE",
+        "THEME",
+    ]);
+    expect.verifySteps(["render_public_asset"]);
+
+    await click(".o-website-snippetsmenu .o-snippets-tabs span:contains(THEME)");
+    await animationFrame();
```

Comment:
Maybe you want to verify that something changed here and that you are on the "theme" tab; if for a reason or another you stayed on the "block" tab, then "render_public_asset" will not be called either without your fix. In fact, maybe that this can be a test in its own ?